modules/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a new custom Ingress Controller. Because the default Ingress Controller might be updated during {product-title} upgrades, creating a custom Ingress Controller can be helpful when maintaining a configuration manually that persists across cluster upgrades.
+
+This example provides a minimal spec for a custom Ingress Controller. To further customize your custom Ingress Controller, see "Configuring the Ingress Controller".
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a YAML file that defines the custom `IngressController` object:
++
+.Example `custom-ingress-controller.yaml` file
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+    name: <custom_name> <1>
+    namespace: openshift-ingress-operator
+spec:
+defaultCertificate: <2>
+    name: custom-ingress-custom-certs
```

Comment:
```suggestion
defaultCertificate:
    name: custom-ingress-custom-certs <2>
```

Below, you have "Set `spec.defaultCertificate` to the name of the secret with the custom wildcard certificate." - seems like this is the line they would change it on?